src/proto/grpc/testing/control.proto

```diff
@@ -37,17 +37,32 @@ package grpc.testing;
 enum ClientType {
   SYNC_CLIENT = 0;
   ASYNC_CLIENT = 1;
+  CLIENT_VARIANT_1 = 2;
```

Comment:
I agree with the above, especially now that we are adding an "Unsupported" status code response. Your suggestion will help keep things more readable. Do you have any API modes to add for C# ? @louiscryan - are there any ClientType/ServerType API variants that you'd like to name in these enums?